dm-manager/pkg/grpc/grpc_server/grpc_server.go

```diff
@@ -154,8 +156,12 @@ func (dms *DeviceManagementService) ReportAMTStatus(
 			err = dms.updateHost(ctx, hostInv.GetTenantId(), hostInv.GetResourceId(),
 				&fieldmaskpb.FieldMask{Paths: []string{
 					computev1.HostResourceFieldAmtSku,
+					computev1.HostResourceFieldPowerStatus,
+					computev1.HostResourceFieldPowerStatusIndicator,
 				}}, &computev1.HostResource{
-					AmtSku: status.AMTStatusEnabled.Status,
+					AmtSku:               status.AMTStatusEnabled.Status,
+					PowerStatus:          UpdateDefaultPowerStatus(hostInv),
```

Comment:
We can make this function local to only this file, since it is only used here, by changing it to `updateDefaultPowerStatus`